website/static/js/osfHelpers.js

```diff
@@ -394,6 +394,67 @@ ko.bindingHandlers.anchorScroll = {
 };
 
 /**
+ * Adds an expansion toggle for too tall items
+ * Example:
+ * <span data-bind='toggleHeight : { height : 100}'></span>
+ */
+ko.bindingHandlers.toggleHeight = {
+    init: function(elem, valueAccessor) {
+        var height = valueAccessor().height || 50;
```

Comment:
Can you separate this code into a plugin? It can go in its own module within the `js/` folder. Use the canonical jQuery plugin style; you can use [folderpicker.js](https://github.com/CenterForOpenScience/osf.io/blob/5254a0d653cf2a80768400e5b86c8d50f3d482d3/website/static/js/folderpicker.js) as an example.

Dong so will allow us to integrate this with libraries/frameworks outside of Knockout in the future.